apps.js

```diff
@@ -15,13 +17,22 @@ app.get('/api/categories', selectAllCategories);
 
 app.get('/api/reviews', selectAllReviews);
 
-app.use('*', (request, response, next) => {
-    response.status(404).send({ msg: '404 Path Not Found'})
+app.get('/api/reviews/:review_id', selectReviewById);
+
+app.use((error, request, response, next) => {
+    if (404 && 'No user found for review_id') {
```

Comment:
I can see that your Promise.reject({status: 404, msg: 'No user found for review_id'}) in your review models has the keys status and msg. Instead of your current if on 23 - you need to be more specific in your if statement but also be able to handle your Promise.rejects dynamically i.e. if(error.status && error.msg){response.status(error.status).send({msg: error.msg}).